Sources/BraintreeCore/BTShopperInsightsRequest.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+/// Buyer data required to use the Shopper Insights feature.
+/// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+@objcMembers public class BTShopperInsightsRequest: NSObject {
```

Comment:
General q based on [this conversation](https://github.com/braintree/braintree_ios/pull/1142/files#r1414559278). I know we've also talked about moving away from the dot syntax and using the init only in v7. Do we have any interest in doing a similar trial of that here and making these properties internal and only the init public? The init is currently internal.